jog_arm/include/jog_arm/jog_arm_server.h

```diff
@@ -220,7 +220,9 @@ class JogCalcs
   Eigen::VectorXd scaleJointCommand(const jog_msgs::JogJoint& command) const;
 
   Eigen::MatrixXd pseudoInverse(const Eigen::MatrixXd& J) const;
+  Eigen::MatrixXd pseudoInverse(const Eigen::MatrixXd& U_matrix, const Eigen::MatrixXd& V_matrix, const Eigen::MatrixXd& S_diagonals) const;
```

Comment:
I'm gonna be strict here, because the MoveIt! maintainers are extremely strict... these should be lowercase, like u_matrix.

Reply:
Have you seen the ROS C++ style guide? http://wiki.ros.org/CppStyleGuide